docs/http.rst

```diff
@@ -91,6 +91,7 @@ Unless otherwise specified, all calls return the following error codes:
    - errno 108 - Router type is invalid
    - errno 110 - Invalid crypto keys specified
    - errno 111 - Missing Required Header
+   - errno 112 - Invalid TTL header value
```

Comment:
I think it'll be ok to call it out on each header, then we can more easily sort by errno on our logging backend. Otherwise we have to be super careful to never change the text at all, or we won't be able to easily match them all for analytics. It also means we can include a rich section for each required header (or link to the MDN page for them) so that there's no chance of confusion by a developer.